### DISCUSSION TITLE
Using external API models in Modelfiles

### DISCUSSION THREAD

**Discussion by aswny** [open-webui/open-webui#306] [Q&A]
Is it possible to use external API, eg OpenAI, OpenRouter API, models in Modelfiles?

Just to be clear, I want to do something like this:
```
FROM openrouter:mistral/mistral-7b
SYSTEM """
You are a helpful assistant...
"""
```

**Comment by tjbck** [comment]
Hi! Not at the moment, but I'm thinking of ways to implement this, stay tuned!

**Reply by aswny** [reply]
Hi @tjbck 

Any updates on this? This feature would be really helpful for me. 

Please let me know when you have any PRs on this. I can help testing out the changes. 

Many thanks.

**Reply by tjbck** [reply]
I've been busy with my day job and haven't had the time to work in it yet unfortunately, I'll definitely keep you updated once I start actively working on it!

https://github.com/ollama-webui/ollama-webui/discussions/602

By the way, we're discussing our new name for this project! Once we decide on the name, we'll make sure to support creating custom model agents for various LLM runners!

**Reply by aswny** [reply]
Refer to the discussion on #665

**Comment by JOduMonT** [comment]
ahah; 
I just asked the same question in Discord
![image](https://github.com/open-webui/open-webui/assets/5204724/8c5f8e22-7279-47fc-87c8-97874082f52d)

**Reply by Yanyutin753** [reply]
can use [one-api](https://github.com/songquanpeng/one-api) to resolve this problem

**Comment by Yanyutin753** [comment]
can use [one-api](https://github.com/songquanpeng/one-api) to resolve this problem
